README.md

```diff
@@ -1,8 +1,10 @@
-[![Build Status](https://travis-ci.org/kadikraman/draftjs-md-converter.svg?branch=master)](https://travis-ci.org/kadikraman/draftjs-md-converter)
+[![Build Status](https://travis-ci.org/FormidableLabs/react-native-app-auth.svg?branch=master)](https://travis-ci.org/FormidableLabs/react-native-app-auth)
 [![npm version](https://badge.fury.io/js/react-native-app-auth.svg)](https://badge.fury.io/js/react-native-app-auth)
 
 # React Native App Auth
 
+### This is the API documentation for `react-native-app-auth >= 2.0.` See version `1.x` documentation [here](https://github.com/FormidableLabs/react-native-app-auth/tree/v1.0.1).
```

Comment:
Can you make the link text the whole "See version `1.x` documentation here" sentence for accessibility reasons?